RELEASE/scripts/autoscend/combat/auto_combat_default_stage2.ash

```diff
@@ -214,7 +214,15 @@ string auto_combatDefaultStage2(int round, monster enemy, string text)
 			}
 			else if(index_of(combatAction, "item") == 0)
 			{
-				handleTracker(enemy, to_item(substring(combatAction, 5)), "auto_replaces");
+				if(contains_text(combatAction, ","))
```

Comment:
Could this look for ",none"? Don't know exact string. Thinking to make this search for a bit more. Prevent potential future issues if a combat item replacer has a comma in the name

Reply:
Good call